packages/salesforcedx-vscode-apex-replay-debugger/src/index.ts

```diff
@@ -243,14 +243,14 @@ function getDialogStartingPath(): vscode.Uri | undefined {
     const lastOpenedLogFolder = extContext.workspaceState.get<string>(
       LAST_OPENED_LOG_FOLDER_KEY
     );
-    if (lastOpenedLogFolder && pathExists.sync(lastOpenedLogFolder)) {
+    if (lastOpenedLogFolder && pathExists.pathExistsSync(lastOpenedLogFolder)) {
```

Comment:
With the update of the newer `pathExist` library, `sync()` was replaced with `pathExistSync()`